packages/core/src/domain/tracekit/computeStackTrace.ts

```diff
@@ -28,11 +29,11 @@ export function computeStackTrace(ex: unknown): StackTrace {
     stack,
   }
 }
+const url = '((?:file|https?|blob|chrome-extension|native|eval|webpack|<anonymous>|\\w+\\.|\\/).*?)'
+const position = '(?::(\\d+))'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:**

- make clearer url and position of what
- avoid duplicating position structure‏

```suggestion
const fileUrl = '((?:file|https?|blob|chrome-extension|native|eval|webpack|<anonymous>|\\w+\\.|\\/).*?)'
const filePosition = '(?::(\\d+))?(?::(\\d+))?'
```

Reply:
`filePosition` contains `?` in-between each position. In the use case of `CHROME_EVAL_RE` this char doesn't exist, therefore not making it composable. Happy to however use the prefix `file` to the variable names

Reply:
oh ok, missed this one, nice then